File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.py
#map함수를 사용하여 input()을 공백을 기준으로 나눈 문자열을 int형식으로 형 변환
n, x = map(int, input().split())
#map함수를 사용하여 공백을 기준으로 나눈 문자열을 int형식으로 구성된 list로 변환
n_list = list(map(int, input().split()))
#n_list순회하면서 입력받은 x가 i보다 크면 i출력 end = ' '을 사용하여 개행하지 않고 한줄로 출력
for i in n_list:
    if x > i:
        print(i, end = ' ')

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.ArrayList;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] str = br.readLine().split(" ");
        int N = Integer.parseInt(str[0]);
        int X = Integer.parseInt(str[1]);
        String[] str1 = br.readLine().split(" ");
        int[] arr = new int[str1.length];
        for(int i = 0; i< str1.length; i++){
            arr[i] = Integer.parseInt(str1[i]);
        }
        //ArrayList동적배열 생성
        ArrayList<Integer> answer = new ArrayList<>();
        for(int i = 0; i < str1.length; i++){
            if (arr[i] < X){
                answer.add(arr[i]);
            }
        }
        for(int i = 0; i< answer.size(); i++){
            System.out.print(answer.get(i) + " ");
        }
    }
}